### PR TITLE
MAUI-3167 API reference links updated for tooltip and legend customization

### PR DESCRIPTION
**Pull request by TamilarasanGunasekaran**
Hi @vigneshrameshsync,
I have updated the API reference links for tooltip and legend customization related UG document.

### CODE REVIEW

File: MAUI/Maps/tooltip.md [hunk 1]
```diff
@@ -382,9 +382,9 @@ public class Model
 ![Maps tooltip settings](images/tooltip/tooltip-appearance-customization.png)
 
 N>
-* Refer to the `ShapeTooltipSettings` to customize the shape tooltip appearance.
-* Refer to the `BubbleTooltipSettings` to customize the bubble tooltip appearance.
-* Refer to the `MarkerTooltipSettings` to customize the marker tooltip appearance.
+* Refer to the [`ShapeTooltipSettings`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapShapeLayer.html#Syncfusion_Maui_Maps_MapShapeLayer_ShapeTooltipSettings) to customize the shape tooltip appearance.
+* Refer to the [`BubbleTooltipSettings`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapShapeLayer.html#Syncfusion_Maui_Maps_MapShapeLayer_BubbleTooltipSettings) to customize the bubble tooltip appearance.
+* Refer to the [`MarkerTooltipSettings`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapLayer.html#Syncfusion_Maui_Maps_MapLayer_MarkerTooltipSettings) to customize the marker tooltip appearance.
```
Comment (vigneshrameshsync) [inline MAUI/Maps/tooltip.md:387]: API links not included in the below NOTE section

N> The `BindingContext` of the Tooltip will be the `MapTooltipInfo`, and it has the `DataItem` property. This property holds the corresponding underline object for the shape, bubble tooltip, and the corresponding `MapMarker` for the marker tooltip.

File: MAUI/Maps/tooltip.md [hunk 2]
```diff
@@ -273,9 +273,9 @@ The [`ShowMarkerTooltip`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Ma
...
-* **Padding** - Change the padding of the tooltip text in maps using the `MaptooltipSettings.Padding` property.
-* **TextStyle** - Change the tooltip text appearance such as `TextColor`, `FontSize`, `FontAttributes`, and `FontFamily` in maps using the `MapTooltipSettings.TextStyle` property.
+* **Background** - Change the background color of the tooltip in maps using the [`MapTooltipSettings.Background`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapTooltipSettings.html#Syncfusion_Maui_Maps_MapTooltipSettings_Background) property.
+* **Padding** - Change the padding of the tooltip text in maps using the [`MaptooltipSettings.Padding`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapTooltipSettings.html#Syncfusion_Maui_Maps_MapTooltipSettings_Padding) property.
+* **TextStyle** - Change the tooltip text appearance such as `TextColor`, `FontSize`, `FontAttributes`, and `FontFamily` in maps using the [`MapTooltipSettings.TextStyle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapTooltipSettings.html#Syncfusion_Maui_Maps_MapTooltipSettings_TextStyle) property.
```
Comment (vigneshrameshsync) [inline MAUI/Maps/tooltip.md:278]: API link not provided for

`TextColor`, `FontSize`, `FontAttributes`, and `FontFamily`

File: MAUI/Maps/tooltip.md (hunk 1 above)
Comment (TamilarasanGunasekaran) [reply MAUI/Maps/tooltip.md:387]: I have included.

File: MAUI/Maps/tooltip.md (hunk 2 above)
Comment (TamilarasanGunasekaran) [reply MAUI/Maps/tooltip.md]: I have provided the API link for it.

**Review by vigneshrameshsync — APPROVED** [review]
The changes are fine.

**Pull request merged**